applications/data_analysis/climatology.ipynb

```diff
@@ -0,0 +1,3551 @@
+{
```

Comment:
<p>This looks really good overall!  Nice work Andy!</p><p> </p><p>I think we could potentially add a little bit more narrative at some point and/or links out to other content, but overall this looks really good and like a great start.  </p><p> </p><p>Some minor comments inline and a question about the final section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>